src/schema/rules/sidecars.yml

```
@@ -6,6 +6,189 @@
 
 
 ---
+# PET common metadata fields
+PETScannerHardware:
+    selectors:
+    - modality == "PET"
+    fields:
+        Manufacturer:
+            level: REQUIRED
+            addendum: Corresponds to DICOM Tag 0008, 0070 Manufacturer.
+        ManufacturersModelName:
+            level: REQUIRED
+            addendum: Corresponds to DICOM Tag 0008, 1090 Manufacturers Model Name.
+        Units:
+            level: REQUIRED
+            addendum: |
+                SI unit for radioactivity (Becquerel) should be used (for example, "Bq/mL").
+                Corresponds to DICOM Tag 0054, 1001 Units.
+        InstitutionName:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0008, 0080 InstitutionName.
+        InstitutionAddress:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0008, 0081 InstitutionAddress.
+        InstitutionalDepartmentName:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0008, 1040 Institutional Department Name.
+        BodyPart:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0018, 0015 Body Part Examined.
+
+PETRadioChemistry:
+    selectors:
+    - modality == "PET"
+    fields:
+        TracerName:
+            level: REQUIRED
+        TracerRadionuclide:
+            level: REQUIRED
+        InjectedRadioactivity:
+            level: REQUIRED
+        InjectedRadioactivityUnits:
+            level: REQUIRED
+        InjectedMass:
+            level: REQUIRED
+        InjectedMassUnits:
+            level: REQUIRED
+        SpecificRadioactivity:
+            level: REQUIRED
+        SpecificRadioactivityUnits:
+            level: REQUIRED
+        ModeOfAdministration:
+            level: REQUIRED
+        TracerRadLex:
+            level: RECOMMENDED
+        TracerSNOMED:
+            level: RECOMMENDED
+        TracerMolecularWeight:
+            level: RECOMMENDED
+        TracerMolecularWeightUnits:
+            level: RECOMMENDED
+        InjectedMassPerWeight:
+            level: RECOMMENDED
+        InjectedMassPerWeightUnits:
+            level: RECOMMENDED
+        SpecificRadioactivityMeasTime:
+            level: RECOMMENDED
+        MolarActivity:
+            level: RECOMMENDED
+        MolarActivityUnits:
+            level: RECOMMENDED
+        MolarActivityMeasTime:
+            level: RECOMMENDED
+        InfusionRadioactivity:
+            level: RECOMMENDED
+            addendum: REQUIRED if PETRadioChemistry.fields.ModeOfAdministration == 'bolus-infusion'
```

```suggestion
            level_addendum: REQUIRED if PETRadioChemistry.fields.ModeOfAdministration == 'bolus-infusion'
```

At least that's what I have in #1017.

---

Ah, then in that case I need to refactor every `addendum` to `level_addendum` then don't I? Will do so on next commit, after this suggestion.

---

Just the ones that should appear in the "Requirement Level" column. The ones that should appear in the "Description" column should still have an "addendum".

---

Ahh whoops.